src/routes/paths.ts

```diff
@@ -79,6 +79,7 @@ export const paths = {
   cosoulArt: (tokenId: string) => `/cosoul/art/${tokenId}`,
   cosoulImage: (tokenId: string) => `/cosoul/image/${tokenId}`,
   cosoulGallery: '/cosoul/gallery',
```

Comment:
should `cosoul/gallery` be live on prod???

Reply:
the debug thing? idk prob not @wingfeatherwave